vulnerabilities/importers/github.py

```diff
@@ -208,7 +209,7 @@ def get_purl(pkg_type: str, github_name: str) -> Optional[PackageURL]:
         vendor, _, name = github_name.partition("/")
         return PackageURL(type=pkg_type, namespace=vendor, name=name)
 
-    if pkg_type in ("nuget", "pypi", "gem", "golang"):
+    if pkg_type in ("nuget", "pypi", "gem", "golang", "npm"):
```

Comment:
Which are the remaining ones we do not support? Do we have an issue to complete this?

Reply:
Follow up issue here https://github.com/nexB/vulnerablecode/issues/1039#issuecomment-1366458885